src/components/ShoppingCart/ListRow.vue

```diff
@@ -219,7 +219,7 @@ export default {
         collectionId: this.collectionId,
         name: this.productName,
         quantity: this.quantity || 1,
-        price: this.customPrice || this.productPrice,
+        price: this.customPrice || this.nftPaymentPriceInUSD,
```

Comment:
|productPrice` is automatically computed as `nftPaymentPriceInUSD` in nft book context, no need to change?